src/rgw/rgw_admin.cc

```diff
@@ -5936,29 +5936,33 @@ int main(int argc, const char **argv)
 
   if (opt_cmd == OPT_LC_LIST) {
     formatter->open_array_section("lifecycle_list");
-    map<string, int> bucket_lc_map;
+    vector<cls_rgw_lc_entry> entries;
     string marker;
 #define MAX_LC_LIST_ENTRIES 100
     if (max_entries < 0) {
       max_entries = MAX_LC_LIST_ENTRIES;
     }
     do {
-      int ret = store->list_lc_progress(marker, max_entries, &bucket_lc_map);
+      int ret = store->list_lc_progress(marker, max_entries, &entries);
       if (ret < 0) {
```

Comment:
will we ever get `-ENOENT` or `-ENODATA`? don't really want to see error if we get these